dev_docs/kibana_server_core_components.mdx

```diff
@@ -0,0 +1,30 @@
+---
+id: kibServerAndCoreComponents
+slug: /kibana-dev-docs/core-intro
+title: Core components
+summary: An introduction to the Kibana server and core components.
+date: 2021-02-26
+tags: ['kibana','onboarding', 'dev', 'architecture']
+---
+
+Core is a set of systems (frontend, backend etc.) that Kibana and its plugins are built on top of.
+
+## Integration with the "legacy" Kibana
```

Comment:
We'll probably want to rewrite some of this eventually, but for now since you're just copying from the existing `README`, it's probably fine to keep as-is.

Reply:
That was my thinking too. It will be a team effort and we'll probably need to set time aside before the new docs system "goes live". For now, I just wanted to add _somewhere_ to house core's documentation.